docs/01_nodeos/03_plugins/trace_api_plugin/index.md

```diff
@@ -1,8 +1,20 @@
 # trace_api_plugin
 
-## Description
+## Overview
 
-The `trace_api_plugin` provides a consumer-focused long-term API for retrieving retired actions and related metadata from a specified block. The plugin defines a new HTTP endpoint accessible (see the [API reference](api-reference/index.md) for more information).
+The `trace_api_plugin` provides a consumer-focused long-term API for retrieving retired actions and related metadata from a specified block. The plugin defines a new HTTP endpoint accessible at `/v1/trace_api/get_block` by the **POST** method and stores serialized block trace data to the filestream for later retrieval via HTTP RPC requests.
```

Comment:
I think we should avoid including API specifics here (like `/v1/trace_api/get_block`) since it's available in the API reference, so it could lead to documentation drift. Also, we should flip the order since the plugin is first configured to store block trace data, and then the HTTP RPC requests can be made. Suggested text:

"The `trace_api_plugin` provides a consumer-focused long-term API for retrieving retired actions and related metadata from a specified block. The plugin stores serialized block trace data to the filesystem for later retrieval via HTTP RPC requests. For more information, see the [Trace API reference](api-reference/index.md)."

Reply:
Removed the endpoint and reversed the sentence order. That's a good suggestion.